src/main.rs

```diff
@@ -1,3 +1,72 @@
+extern crate pancurses;
```

Comment:
It's not obvious to me this line is needed. `extern crate` used to be required for lots of things but doesn't seem so prevalent anymore although I never really understood it. Looking at [the docs](https://doc.rust-lang.org/reference/items/extern-crates.html) it sounds like it just brings it into scope which shouldn't be necessary (somewhat backed up by [this post](https://stackoverflow.com/a/29404692/2891835) although I haven't fully read what they've linked to).

Reply:
You caught me copy/pasting from the [pancurses docs](https://crates.io/crates/pancurses) 😅 

I'll try removing it